setup.py

```diff
@@ -139,7 +138,7 @@ def run(self):
         "sphinx-rtd-theme>=0.5.0,<1.1",
         "sphinxcontrib-spelling==7.*",
     ],
-    "runner": ["cwlref-runner==1.0"],
+    "runner": ["cwlref-runner==1.0", "toil==5.5.0"],
```

Comment:
Should we add this to install requires instead? 
Or if we want to make it optional, add a "toil" extras so one can do `pip install renku[toil]` for renku with toil?

Reply:
yeah actually i was thinking of the latter, i.e. toil extras... but basically i would rename the `runner` then to `cwltool` or `cwl` instead of `runner`

Reply:
I don't think we need cwlref-runner. I've removed the whole `runner` extras in the poetry PR.
All it does is install an executable `cwl-runner` that references `cwltool` so a user can do e.g. `cwl-runner workflow.cwl`. But we don't use it anywhere in code (it used to be used in some CI a long time ago), so I see no reason keeping it.

Reply:
@Panaetius i've added `toil` extras as an optional extra, i.e. it's not part of `all`, hence toil based execution/rerun/update tests are not being tested on CI. should we add to one of the CI jobs toil as well, so that way its being tested?

Reply:
I'd just add it to all as well. The cache is based on `all` and all tests should use the same cache, so that makes sense (and users don't usually do `pip install .[all]`)

Reply:
And don't forget to update the lock file with `poetry.lock`

Reply:
`poetry lock --no-update` updates quite a lot of dependencies. any other suggested way to update the `poetry.lock` or it's fine :)

Reply:
I think that's just toil and all its subdependencies.